spec/features/bullet_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "spec_helper"
+
+RSpec.describe "Bullet" do
+  let(:dev_config) { IO.read("#{project_path}/config/environments/development.rb") }
+  let(:application_job) { IO.read("#{project_path}/app/jobs/application_job.rb") }
+
+  before(:all) do
+    drop_dummy_database
+    remove_project_directory
+    create_dummy_project
+  end
+
+  # rubocop:disable RSpec/MultipleExpectations
```

Comment:
podés sacar esta regla y hacer un it por cada opción. Como el proyecto se corre en before all, no se va a crear un proyecto por it